src/number_format.js

```diff
@@ -128,12 +129,14 @@ class NumberFormat extends React.Component {
   getFloatString(num, props) {
     props = props || this.props;
     const {decimalSeparator, thousandSeparator} = this.getSeparators(props);
-    return num.replace(new RegExp(escapeRegExp(thousandSeparator || ''), 'g'), '').replace(decimalSeparator, '.');
+    const number = num || '';
```

Comment:
This can be moved to arguments 
```js
getFloatString(num='', props)
```

Reply:
Ohh yes, 
Instead of creating new const just change it to
```js
num = num || '';
```
or
```js
return (num || '').replace(new RegExp(escapeRegExp(thousandSeparator || ''), 'g'), '').replace(decimalSeparator, '.');
```